MidasCivil_Adapter/Convert/ToMidasCivil/Elements/FromFEMesh.cs

```diff
@@ -34,10 +37,12 @@ public static partial class Convert
 
         public static string FromFEMesh(this FEMesh feMesh)
         {
-            string midasElement = "";
+        int i = NextFreeId(typeof(FEMesh));
```

Comment:
`NextFreeId` stops it from being built, as discussed perhaps just make a `private` method with the following to work out the next available `id`:
```
                    string section = "ELEMENT";

                    if (ExistsSection(section))
                    {
                        index = GetMaxId(section) + 1;
                    }
                    else
                    {
                        index = 1;
                    }
```

Reply:
everything in the MidasCivilAdapter name space is protected so cant use either of these functions in this space, however I have modified the code to get it to work. Please review the changes in both namespaces.